src/Microsoft.Diagnostics.DebugServices.Implementation/ModuleService.cs

```diff
@@ -318,7 +326,7 @@ internal byte[] GetBuildId(ulong address)
         /// <returns>version string or null</returns>
         protected string GetVersionString(ulong address)
         {
-            Stream stream = RawMemoryService.CreateMemoryStream();
+            Stream stream = MemoryService.CreateMemoryStream();
```

Comment:
The version string is never used in the that recursive path and it calls SearchVersionString that is used to find the string uses the MemoryService instead of the raw one anyway.